PolyEngine/Core/Src/String.cpp

```diff
@@ -126,11 +130,10 @@ String String::Replace(char what, char with) const {
 	return s;
 }
 
-String String::Replace(const String& /*what*/, const String& /*with*/) const {
-	ASSERTE(false, "Not implemented yet.");
-	//split source by delimiter a
-	//return string joined with separator b
-	return String();
+String String::Replace(const String& what, const String& with) const {
+	String s = "";
+	Dynarray<String> splitted = this->Split(what);
+	return s.Join(splitted.GetData(), splitted.GetSize(), with);
```

Comment:
Join is a static method. Remove variable `s` and simply call `return Join(splitted.GetData(), splitted.GetSize(), with);`